cves/CVE-2016-6186.yml

```diff
@@ -169,8 +181,10 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: |
+    The vulnerability was in the ui component because of the fact that this input field
+    that requires sanitization may be found in the frontend component.
+  name: User Interface
```

Comment:
UI is fairly generic. Is this part of a specific subsystem within the UI?

Reply:
Specified the control inputs component of the UI